src/handlers.rs

```diff
@@ -146,6 +147,12 @@ fn test_resolve_module() {
       add_root!("/Users/rld/src/deno/hello.js"),
       add_root!("/Users/rld/src/deno/hello.js"),
     ),
+    (
+      add_root!("/Users/rld/src/deno/hello.js"),
+      add_root!("/Users/rld/src/deno/hello.js"),
```

Comment:
not that it really matters in this case - but the containing_file should be different - so change this line to something_else.js

Reply:
Fixed it.
(Although - it's not technically impossible for a module to resolve itself? 
Even if we don't allow it, in order to detect that a module is trying to import itself, we'd have to run it through the resolver first.)